### PR TITLE
Accelerarted markets data

### PR DESCRIPTION
**Pull request by ifyio**
cc/ @commuterjoy 

Initial dirty mvp

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "no-var": 1,
     "space-before-function-paren": [1, "always"],
     "strict": [1, "global"],
-    "no-console": 1,
```
Comment (commuterjoy) [inline .eslintrc.json]: :/

File: client/components/markets-data-details/main.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	document.querySelector('.js-markets-data-details').innerHTML = marketsDataDetails.render({markets: model.markets});
+	marketsDataDetails.init();
+	Tabs.init(document.querySelector('.js-markets-data-details'), {
+		disablefocus: false,
+		disableFocus: false
+	});
+}
+
+function handleError (error) {
+	console.error(error);
```
Comment (commuterjoy) [inline client/components/markets-data-details/main.js]: not in production code please

File: client/libs/markets-data-api/securities.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import crossDomainFetch from 'o-fetch-jsonp';
+
+export function fetchQuotes (symbols) {
+
+	return crossDomainFetch(`https://next-markets-proxy.ft.com/securities/v1/quotes?symbols=${symbols}`, { timeout: 5000 })
```
Comment (commuterjoy) [inline client/libs/markets-data-api/securities.js]: 5000 is generous! 2000ms would be a max IMHO.

Comment (ifyio) [inline client/libs/markets-data-api/securities.js]: agreed

File: .eslintrc.json (hunk 1 above)
Comment (ifyio) [inline .eslintrc.json]: wanted to keep a console.log in to test some things. Planned to change it back

File: client/components/markets-data-details/main.js (hunk 2 above)
Comment (ifyio) [inline client/components/markets-data-details/main.js]: ok no prob

**Pull request merged**